app/handler/MessageEvent/handler.py

```diff
@@ -0,0 +1,102 @@
+import os
+import requests
+import datetime
+import traceback
+from pyquery import PyQuery as pq
+from linebot.models import (
+    TextSendMessage,
+    CarouselTemplate, CarouselColumn, PostbackAction,
+    MessageAction, TemplateSendMessage, URIAction
+)
+from app.line import line_bot_api, line_handler
+
+class MessageEventHandler:
+    def __init__(self):
+        pass
+
+    def handle(self, event):
+        try:
+            text = event.message.text.strip()
+
+            if (text.lower().startswith("!today")):
+                self.featureToday(event)
+        except Exception as error:
+            line_bot_api.reply_message(
+                event.reply_token,
+                TextSendMessage(text="Error, silahkan coba lagi")
+            )
+            print(str(error))
+            traceback.print_exc()
+
+    def featureToday(self, event):
```

Comment:
Sepertinya convention untuk python, nama variable dan function pakai lower_camel_case

Reply:
Sepertinya saya juga ada khilaf gini di beberapa tempat wkwkwkw

Reply:
lower snake case deng namanya

Reply:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.

Variable names follow the same convention as function names.

Ini gan kata peps-008

Reply:
woke woke